src/lib/annotations/doc/DocDrawingDialog.js

```diff
@@ -0,0 +1,11 @@
+import AnnotationDialog from '../AnnotationDialog';
+
+class DocDrawingDialog extends AnnotationDialog {
+    addAnnotation() {}
```

Comment:
Docs for these functions? And maybe for the class

Reply:
Will add them